inc/saitunnel.h

```diff
@@ -647,6 +676,35 @@ typedef enum _sai_tunnel_attr_t
      */
     SAI_TUNNEL_ATTR_TERM_TABLE_ENTRY_LIST,
 
+    /**
+     * @brief Tunnel EXP mode (pipe or uniform model)
+     *
+     * @type sai_tunnel_exp_mode_t
+     * @flags CREATE_ONLY
+     * @default SAI_TUNNEL_DSCP_MODE_UNIFORM_MODEL
+     */
+    SAI_TUNNEL_ATTR_ENCAP_EXP_MODE,
+
+    /**
+     * @brief Tunnel EXP value (3 bits)
+     *
+     * @type sai_uint8_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @condition SAI_TUNNEL_ATTR_ENCAP_EXP_MODE == SAI_TUNNEL_EXP_MODE_PIPE_MODEL and SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_MPLS
+     */
+    SAI_TUNNEL_ATTR_ENCAP_EXP_VAL,
+
+    /**
+     * @brief Tunnel EXP mode (pipe or uniform model)
+     *
+     * Default SAI_TUNNEL_EXP_MODE_UNIFORM_MODEL
+     *
+     * @type sai_tunnel_exp_mode_t
+     * @flags CREATE_ONLY
+     * @default SAI_TUNNEL_DSCP_MODE_UNIFORM_MODEL
```

Comment:
L704 SAI_TUNNEL_DSCP_MODE_UNIFORM_MODEL should be SAI_TUNNEL_EXP_MODE_UNIFORM_MODEL

Reply:
yes, will fix the mode